docs/index.bs

```diff
@@ -1605,15 +1605,27 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
       The <dfn method for="InstallEvent"><code>addRoutes(|rules|)</code></dfn> method steps are:
 
-        1. Let |serviceWorker| be the [=current global object=]'s associated [=ServiceWorkerGlobalScope/service worker=].
-        1. Let |routerRules| be a copy of |serviceWorker|'s [=list of router rules=].
+        1. Let |event| be [=this=].
+        1. If |event|'s [=dispatch flag=] is unset, [=throw=] an "{{InvalidStateError}}" {{DOMException}}.
+        1. If |rules| is not either of a {{RouterRule}} dictionary or a sequence of {{RouterRule}} dictionaries, then return [=a promise rejected with=] a {{TypeError}}.
```

Comment:
This is unnecessary as the IDL layer ensures it will always be true.

Reply:
good catch!
removed.